zproject/prod_settings_template.py

```diff
@@ -273,12 +273,13 @@
 ## "APIs & Services" > "Credentials", and create a "Project" which will
 ## correspond to your Zulip instance.
 ##
-## (2) Navigate to "APIs & services" > "Library", and find the
-## "Identity Toolkit API".  Choose "Enable".
+## (2) Go to "Oauth consent screen" and create a consent screen,
+## authorizing your domain and enabling the .../auth/userinfo.email,
+## .../auth/userinfo.profile and openid scopes.
 ##
 ## (3) Return to "Credentials", and select "Create credentials".
-## Choose "OAuth client ID", and follow prompts to create a consent
-## screen.  Fill in "Authorized redirect URIs" with a value like
+## Choose "OAuth client ID", and follow prompts.
```

Comment:
I'm not sure what we mean concretely by "follow prompts" here.

Reply:
@timabbott Yeah you're right, kinda awkward phrasing. Rewrote this sentence to be more specific.

Plus added a mention (in oauth screen creation step) about using the "Internal" user type as something likely useful to Google Workspace users.